code/game/gamemodes/meteor/meteors.dm

```diff
@@ -468,7 +468,9 @@ var/list/blob_candidates = list()
 
 	playsound(user, 'sound/weapons/rocket.ogg', 100)
 
-	new projectile_type(get_turf(src), get_turf(A))
+	var/obj/item/projectile/meteor/in_chamber = new projectile_type(get_turf(src), get_turf(A))
+
+	log_attack("[user.name] ([user.ckey]) fired \the [src] (proj:[in_chamber.name]) at [A] [ismob(A) ? "([A:ckey])" : ""] ([A.x],[A.y],[A.z])" )
```

Comment:
It is better to use add_logs() for this - keeps the formatting consistent and stops you using COLONS